source/includes/_rendering.md

```diff
@@ -178,4 +177,94 @@ function myRenderOverlay() {
 
 Here we are making all of our rendered objects be perfectly aligned horizontally on the screen for all windows sizes. We start off by getting the height of the current window, with the call to `Renderer.screen.getWidth()`.
 
-Then, for each part we render, we get half the width of the window, and then subtract half the width of our rendered object. For a string, this is done with `(renderWidth / 2) - (Renderer.getStringWidth(textToRender) / 2)`. For a fixed width object, you can replace `Renderer.getStringWidth(textToRender)` with the width of the object. Notice how we use the `Text` object and instantiate it outside of the render function. The text object allows you to set additional properties, such as shadow and color. 
\ No newline at end of file
+Then, for each part we render, we get half the width of the window, and then subtract half the width of our rendered object. For a string, this is done with `(width / 2) - (Renderer.getStringWidth(textToRender) / 2)`. For a fixed width object, you can replace `Renderer.getStringWidth(textToRender)` with the width of the object. Notice how we use the `Text` object and instantiate it outside of the render function. The `Text` object allows you to set additional properties, such as shadow and color.
+
+# Tessellator
+
+The Tessellator object is meant for rendering in 3 dimensions. You have to call this object under the `renderWorld` trigger.
+
+## Drawing Text
+
+> You can set floating text at specific world coordinates using `Tessellator.drawString(text, x, y, z)`
+
+```javascript
+register("renderWorld", myWorldRender);
+
+function myWorldRender() {
+  Tessellator.drawString("Hello there!", 100, 100, 100);
+}
+```
+
+This will draw a the words "Hello There" at the world location 100, 100, 100.
+You can also change the text color, whether or not to render the shadow, change the scale, and whether or not
+the size increases from distance. All of these examples below would work also.
+
+```javascript
+Tessellator.drawString("Blue text!", 100, 100, 100, Renderer.BLUE);
+Tessellator.drawString("Green text with shadow!", 100, 100, 100, Renderer.GREEN, true);
+Tessellator.drawString("Bigger red text!", 100, 100, 100, Renderer.RED, false, 2);
+Tessellator.drawString("Hello there!", 100, 100, 100, Renderer.DARK_PURPLE, true, 2, false);
+```
```

Comment:
No need to spell out every possible method invocation, as that is what the javadocs are for. The slate only has to be a general overview.

```suggestion
This will draw the words "Hello There" at the world location `(100, 100, 100)`.
You can also change the text color and scale, whether or not to render the shadow, and whether or not
the size increases from distance. 
```